### PR TITLE
Release v0.1.50

### PR DESCRIPTION
**Pull request by Vincent056**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,22 +89,6 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 -
 
-## [0.1.50] - 2022-04-05
```
Comment (rhmdnd) [inline CHANGELOG.md:61]: I didn't catch this in the previous review [0].

I was wondering why this what already in the file when we hadn't release v0.1.50, yet. I think we should be OK now if the Makefile target generated these notes.

[0] https://github.com/ComplianceAsCode/compliance-operator/commit/0dfd6115990a3c6f6b0df39a906e4f97ebca6946

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,22 +89,6 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 -
```
Comment (rhmdnd) [inline CHANGELOG.md:90]: nit: I think we can remove the empty sections. We've done that with past releases.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +47,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   resources for the `gitopsservices.pipelines.openshift.io` API group. This is
   necessary to automate checks that ensure the cluster is using GitOps operator.
 
+- Added necessary permessions for api-resource-collector so that the new rule
+  `cluster_logging_operator_exist` can be evaluated properly. [1]
+  [1] https://github.com/ComplianceAsCode/content/pull/8511
+
 ### Fixes
 
 - The compliance content images have moved to
```
Comment (rhmdnd) [inline CHANGELOG.md:56]: I think we should update the Customer Case below so that it references a public bugzilla or an issue in this new repository.

**Review by rhmdnd — CHANGES_REQUESTED** [review]
Just a couple minor comments on release notes. We can probably fix those up in a separate patch and then rebase this so that they're separate.

**Comment by rhmdnd** [comment]
I think we want to propose this to the CaC/compliance-operator fork now.

**Comment by Vincent056** [comment]
Close this one to OpenPR against CaC/Compliance-Operator repo

**Pull request closed without merging**